### PR TITLE
WIP Fix flicker when user mouses over the very leftmost edge of party member avatar

### LINKED ISSUE #10379: Flicker when user mouses over the very leftmost edge of party member avatar
[//]: # (Before logging this issue, please post to the Report a Bug guild from the Habitica website's Help menu. Most bugs can be handled quickly there. If a GitHub issue is needed, you will be advised of that by a moderator or staff member -- a player with a dark blue or purple name. It is recommended that you don't create a new issue unless advised to.)

[//]: # (Bugs in the mobile apps can also be reported there.)

[//]: # (If you have a feature request, use "Help > Request a Feature", not GitHub or the Report a Bug guild.)

[//]: # (For more guidelines see https://github.com/HabitRPG/habitica/issues/2760)

[//]: # (Fill out relevant information - UUID is found from the Habitia website at User Icon > Settings > API)
### General Info
  * UUID: Various
  * Browser: Chrome, others?
  * OS: Windows 10, others?

### Description
[//]: # (Describe bug in detail here. Include screenshots if helpful.)

There is a small area (1-2 pixels wide perhaps?) in the party header where if the user mouses over it, it flickers rapidly and unpleasantly between displaying and hiding the first party member's mouseover details.

Video: https://drive.google.com/file/d/1q7ONERmRuZXTo5VVq6GQ9sv_DHUAQSJj/view

### PR DESCRIPTION
**Pull request by haydenbetts**
Fixes #10379 

### Changes
Add one pixel of margin-right to the .member-stats class.

Video Before: https://drive.google.com/file/d/1q7ONERmRuZXTo5VVq6GQ9sv_DHUAQSJj/view
Video After: https://drive.google.com/open?id=1FSM-PFIjGzr1yEvs6OkjQ2vcQaUKd5i6

### CODE REVIEW

**Comment by haydenbetts** [comment]
@paglias @SabreCat @TheHollidayInn How do I run the client side unit tests (those in `/test/client/unit`) locally? I don't see any documentation about it on the wikia.

`npm run test:client:unit` gives me:
`4 verbose stack Error: missing script: test:client:unit`

Also - do people usually add client-side tests in order to pass `coverage/coveralls` for PRs like this one?

**Comment by paglias** [comment]
non run client:unit is the command to use!

File: test/client/unit/specs/components/memberDetails.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import Vue from 'vue';
```
Comment (paglias) [inline test/client/unit/specs/components/memberDetails.js:1]: there are several linting errors here https://travis-ci.org/HabitRPG/habitica/jobs/384875487

**Review by paglias — CHANGES_REQUESTED** [review]

**Comment by haydenbetts** [comment]
@paglias - I am having some trouble mounting the `MemberDetailsComponent` for unit testing.

Among other errors I am getting:
`Missing required prop: "member"` when I run `npm run client:unit`

Is this error occurring because in the [MemberDetailsComponent](https://github.com/HabitRPG/habitica/blob/develop/website/client/components/memberDetails.vue), the  `member` object would usually be fetched from the store, and my unit tests do not have access to the store? Would it be possible to set a default `member` prop directly with some sample data for unit testing? Or do I need to simulate a store somehow?

Note: I know this is a of questions for a PR that adds one px of margin to something but hopefully the learning will be useful for future contributions!

**Comment by paglias** [comment]
@haydenbetts no worries about questions :) Yeah it looks like you need to pass the `member` prop to the component in tests, you can see some examples of passing props to tests here https://github.com/HabitRPG/habitica/blob/develop/test/client/unit/specs/components/categories/categoryTags.js#L9 but I think in this case this may be the easiest way: https://github.com/HabitRPG/habitica/blob/develop/test/client/unit/specs/components/ui/drawer.js#L13

**Comment by SabreCat** [comment]
Setting up a client test for 1px margin of CSS seems like overkill, here. @paglias @TheHollidayInn what do you think about removing the test and merging just the layout fix?

**Comment by paglias** [comment]
@SabreCat I'm fine with removing the test and merging the pr!

**Comment by SabreCat** [comment]
@haydenbetts We're on a temporary hold for merges into `develop` this week because of a breaking change we're working on, but this will get merged once our staging environment is out of code freeze. Do you have a Habitica UUID we can award contributor credit to?

**Pull request merged**